src/Bridge/Doctrine/Persister/ObjectManagerPersister.php

```diff
@@ -130,8 +143,11 @@ private function getPersistableClasses(ObjectManager $manager): array
     private function getMetadata(string $class): ClassMetadata
```

Comment:
And here, you can inject the ObjectManager and get metadata directly with `$objectManager->getMetadata($class)`